app/course/controllers/CourseCtrl.js

```diff
@@ -302,7 +302,8 @@ class CourseCtrl {
 
       // Will update the sectionGroup plannedSeats using section seats
       // If the section is numeric based (example: 'PSC 040 - 001')
-      if (isNumber(section.sequenceNumber) == true) {
+      // or if their is only one section in the group
+      if (isNumber(section.sequenceNumber) == true || sectionGroup.sections.length == 1) {
```

Comment:
This boolean expression can be rewritten as
```js
isNumber( section.sequenceNumber ) || sectionGroup.sections.length === 1
```
Using `===` is a good habit because type is also considered when comparing two values. We want to avoid situations where `'1' == 1` returns true when we want it to return false. Try it in the console! It's crazy!